### DISCUSSION TITLE
Logo Discussion!

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#168] [Polls]
# 1.
![logo-mac](https://github.com/user-attachments/assets/072bbd7c-6b69-43eb-b3d1-6c6b402620bf)
# 2.
![image](https://github.com/user-attachments/assets/0d5a5d3c-d387-4044-ac9f-7efe6e825717)
# 3.
![image](https://github.com/user-attachments/assets/9f037a74-8003-4a48-b0ce-7a448ce7adc1)

**Comment by TheBestUser01234567890** [comment]
logo 3 is actually 2 logos.
also, I personally like this logo:
![image](https://github.com/user-attachments/assets/7e1c6aa7-128e-4986-9f4e-3b097560b53e)

**Reply by mr-cheffy** [reply]
It's separated by themes :P, we can then discuss and change the logos once we find one we like

**Reply by TheBestUser01234567890** [reply]
> It's separated by themes :P, we can then discuss and change the logos once we find one we like

what do you mean it's separated by themes

**Reply by mr-cheffy** [reply]
Different artists gave different kinds of logos, right now we are just deciding what "logo kind" we are choosing

**Reply by mayemu** [reply]
with logo 3 maybe you could have a different color logo based on the color scheme the user picked? haha not sure how possible that would be. either way logo 3 all the way for me

**Reply by FabiKel** [reply]
The second row of logo 3 would be nice for nightly builds or something like that. 
I also like the first one, but I don't like the background pattern. But without the pattern it looks too boring, I think.

**Reply by Donnnno** [reply]
The shape is more important at this stage, and we can tweak the colors to be different and more fitting, of course! 

@FabiKel that was my original idea ;) → https://github.com/zen-browser/desktop/issues/150#issue-2447181352

**Comment by Yuzuctus** [comment]
I find the 3 very pretty but when I use Zen I find that the 1st is much more faithful to the reality of the UI so that all the interface and logo are consistent.

**Comment by 1280px** [comment]
The Z letter might get you really, really, really misunderstood nowadays. Just thoughts.

**Reply by Donnnno** [reply]
It depends on what you do with it. This is clearly a different thing.
Don't let fascists take control over a fucking letter. (same goes with x)
